code/modules/mob/mob_helpers.dm

```diff
@@ -16,11 +16,9 @@
 	return 0
 
 /mob/living/carbon/human/isSynthetic()
-	// If they are 100% robotic, they count as synthetic.
-	for(var/obj/item/organ/external/E in bodyparts)
-		if(!(E.status & ORGAN_ROBOT))
-			return 0
-	return 1
+	if(get_species() == "Machine")

```

Comment:
I changed this because having all mechanical external limbs doesn't suddenly make you synthetic; your internal organs would have to be, as well; that said, this presents problems for races that have no internal organs, who would be considered synthetic if all their internal AND external organs were robotic (since the first would return `1` by virtue of them having no internal organs).